src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
```

Comment:
do you need it?

Reply:
oh, i see, it's not necessary, thank You!